infrastructure/src/main/java/mops/database/dto/AntwortDto.java

```diff
@@ -0,0 +1,37 @@
+package mops.database.dto;
+
+import java.util.HashSet;
+import java.util.Set;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.relational.core.mapping.Column;
+import org.springframework.data.relational.core.mapping.Table;
+
+
+@AllArgsConstructor
+@Data
+@Table("antwort")
+public class AntwortDto {
+  @Id
+  Long id;
+  String textantwort;
+  @Column("antwort")
+  Set<AuswahlDto> ausgewaehlt;
```

Comment:
Why did you put a string for text answers AND a set for multiple-choice choices in one class?

Reply:
So we only need this one Table to save the answers. There is not really the need for two seperated Tables 'TextAntwort' and 'MultipleAntwort which would have to extend the 'Antwort' table. We already have enought Dtos, so why add unnecessary ones.
Database handling is also easier this way, but we can make changes if you want

Reply:
No, you don't have to change this, if you think, it is okay. It was only an annotation. But when you translate the DTO to business objects you need a few case distinctions.